.github/workflows/elixir.yml

```diff
@@ -1,21 +0,0 @@
-name: Elixir CI
```

Comment:
It's worth maintaining a reference to this `elixir.yml` file in https://github.com/dwyl/technology-stack/issues/73 because I _agree_ that we need to explore GitHub Actions/Workflows in more detail. 
If we could use GitHub's CI and it worked as expected it would **`reduce`** our need to depend on other tools. 💭 
For now I'm happy to use Travis-CI because it is a superior / more mature product.